deploy/crds/create-br1-linux-bridge.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: nmstate.io/v1
```

Comment:
this is only for testing now, could we keep it in test modules? that will make sure that we keep only manifests that are actually used. if we stock them under crds/ it will get out of control soon.

Reply:
What about deploy/demos ?

Reply:
For demos, yes. Not for test though. Can we keep it in code? Like, have a helper that applies NodeNetworkState with desiredState build from given string? Another helper to check that currentState matches given string yaml.